docs/source/usage/plugins/radiation.rst

```diff
@@ -295,7 +299,8 @@ Memory Complexity
 Accelerator
 """""""""""
 
-each energy bin times each coordinate bin allocates one counter (``float_X``) permanently and on each accelerator.
+locally, ``numTmpResults`` matrices of the size number of frequencies times directions times is permanently allocated.
```

Comment:
I guess `times directions times` was meant to be `times number of directions`?

Reply:
Regarding the memory footprint:
"The memory usage is increased by $N_{omega}N_{directions}$ for each temporary array."
AFAIK we can render LaTeX in the documentation.

Reply:
I agree with @Anton-Le on this sentence.